docs/content/python/guides/enforcement/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Enforcing an Oso Policy (Oso 0.20.0 Alpha)
+weight: 10
+any: true
+# showContentForAnyLanguage: true
+# draft: True
+---
+
+{{% callout "Note: 0.20.0 Alpha Feature" %}}
```

Comment:
Might be good to make this a shortcake as we'll have it on several pages now.

Reply:
Lol 🍰 

Good idea though

Reply:
Can't do this unless we redefine `callout` -- I was surprised to learn that hugo doesn't like using shortcodes from other shortcode definitions :(

https://discourse.gohugo.io/t/error-using-shortcodes-in-template-block-redefinition/22211